src/Mod/Path/PathScripts/PathAdaptiveGui.py

```diff
@@ -214,6 +214,10 @@ def getFields(self, obj):
         obj.setEditorMode('Stopped', 2)  # hide this property
 
 
-Command = PathOpGui.SetupOperation('Adaptive', PathAdaptive.Create, TaskPanelOpPage,
-                                   'Path-Adaptive', QtCore.QT_TRANSLATE_NOOP("PathAdaptive", "Adaptive"),
-                                   QtCore.QT_TRANSLATE_NOOP("PathPocket", "Adaptive clearing and profiling"))
+Command = PathOpGui.SetupOperation('Adaptive',
+        PathAdaptive.Create,
+        TaskPanelOpPage,
+        'Path_Adaptive',
```

Comment:
I assumed that line 219 reflected the name of the icon, is that correct?

Reply:
Yes, that's right.